packages/flutter/lib/src/painting/matrix_utils.dart

```diff
@@ -169,6 +169,66 @@ class MatrixUtils {
     transform = new Matrix4.copy(transform)..invert();
     return transformRect(transform, rect);
   }
+
+  /// Create a transformation matrix which mimics the effects of tangentially
+  /// wrapping the plane on which this transform is applied around a cylinder
+  /// and then looking at the cylinder from a point outside the cylinder.
+  ///
+  /// [radius] simulates the radius of the cylinder the plane is being wrapped
```

Comment:
For arguments, use backticks rather than square brackets.

Reply:
Also, prefer not starting a sentence with a member since then you have to start it in lowercase. I would phrase this as `The 'radius' argument simulates...` (but with backticks, not apostrophes).

Reply:
ah, I always get those mixed up. The dart analyzer in vscode or intellij does seem to correctly link the arguments to their definition in the function signature. Was that your concern?

Reply:
The problem is the current conventions cause issues in classes where arguments match properties. So I'm trying to make sure we're consistent on a new style.
See https://github.com/dart-lang/dartdoc/issues/1486 for details.

Reply:
done